prime-router/src/main/kotlin/azure/DatabaseAccess.kt.orig

```diff
@@ -1,851 +0,0 @@
-package gov.cdc.prime.router.azure
```

Comment:
Why is this file removed? It doesn't seem to have anything to do with linting?

Reply:
.orig files are generated during a merge. These were generated when I updated my master branch and then merged it to this one.